### ISSUE TITLE
[Accessibility]A11y_PowershellEditorServices_Installation_Screenreader:Narrator does not read the drop down list items of  the Disable button and More button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61705]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10
MAS1.3.1
**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Extension" button of "ActivityBar"->Enter "PowerShell" in "Search Extension" text field and click on "Install" button.
3.Navigate through keyboard such as "Tab" key to "More" and "Disable" button.
4.Observed that whether narrator reads the dropdown list item when dropdown is opened.

**Actual:**
Narrator does not read the dropdown list items after click on "More" and "Disable" button.

**Expected:**
Narrator should read the dropdown list items when dropdown is opened.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not able to know the control written under drodown.

**MAS Reference:**
MAS1.3.1, Screenreader
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference**
[2726890_A11y_PowershellEditorServices_Installation_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2510236/2726890_A11y_PowershellEditorServices_Installation_Screenreader.pptx)

**Comment by weinand** [comment]
@AccessibilityTestingTeam-TCS I get the impression that creating these PowerShell issues against an old version of VS Code and in the wrong repository is a waste of your and our time...

**Comment by isidorn** [comment]
Narrator is not supporetd
